rep-2004.rst

```diff
@@ -434,86 +354,318 @@ For example, this might be a tutorial package which is not intended for reuse bu
 Package Requirements
 ~~~~~~~~~~~~~~~~~~~~
 
-*Note: bullets below that start with [ROS Core], will be the prescription for what we do in the core packages in order to meet the associated requirements*
-
 Requirements to be considered a 'Level 4' package:
 
-* Version Policy:
-
-  * No requirements, but having a policy is still recommended (e.g. ``semver``), even if the version is not yet stable (e.g. >= 1.0.0 for ``semver``)
+#. **Version Policy:**
 
-* Change Control Process:
+   #. No requirements, but having a policy is still recommended (e.g. ``semver``), even if the version is not yet stable (e.g. >= 1.0.0 for ``semver``)
 
-  * No explicit change control process required, but still recommended
+#. **Change Control Process:**
 
-* Documentation:
+   #. No explicit change control process required, but still recommended
```

Comment:
this one was hard to map to the chart because it's not recommending any specific step of the change control process, just the whole thing. so that would mean a ● for level 4's `2.i - 2.v`, but `2.ii`, `2.iii` and `2.v` aren't even "recommended" for levels 2 and 3. why would level 4 be more stringent than 2 and 3?

Reply:
Yeah, the point of this was "all things under change control are optional, but recommended", but it isn't precise. I'm not sure what to do here, what do you think we should do?

Reply:
Well, "all things optional, but recommended" is kinda the case for everything right? based on line 59:

> While each quality level will have different requirements, it's always possible to overachieve in certain requirements even if other requirements prevent a package from moving up to the next quality level.

That doesn't mean we should put a dot in every spot on the chart though.

I guess the better question is are any of the steps _highly_ recommended for level 4?
Levels 2 and 3 don't require or even highly recommend having DCO, a peer review policy, or documentation for all change requests. 
So I think it's safe to assume what you're highly recommending is just the first point, all changes must go through a change request.
And so maybe it's a good idea to change this line to:

```suggestion
   #. No explicit change control process required, but it is recommended all changes occur through a change request.
```